NuRadioReco/detector/RNO_G/rnog_detector.py

```diff
@@ -692,11 +750,42 @@ def get_channel_signal_chain(self, station_id, channel_id):
         """
         channel_info = self.__get_channel(
             station_id, channel_id, with_signal_chain=True)
-        return self.__buffered_stations[station_id]["channels"][channel_id]["signal_chain"]
+        return channel_info["signal_chain"]
+
+    @_check_detector_time
+    def get_amplifier_response(self, station_id, channel_id, frequencies):
+        """
+        Returns the complex response function (for the passed frequencies)
+        for the entire signal chain of a channel. I.e., this includes not
+        only the (main) amplifier but also cables and other components.
+
+        Parameters
+        ----------
 
-    @check_detector_time
+        station_id: int
+            The station id
+
+        channel_id: int
+            The channel id
+
+        frequencies: np.array(np.complex128)
```

Comment:
Frequencies are real (and in our case, positive), but I'm just nitpicking again.

Reply:
fix